### ISSUE TITLE
make master branch stable

### ISSUE THREAD

**Issue by FlowerWrong** [qor/qor-example#127]
Hi!  Thanks a lot. Can you make the `master` branch stable? Stable means we can run this demo without error. You can develop it in other branch, eg `dev`.

**Comment by cryptix** [comment]
Why did you close this? I think it is a legitimate request.

**Comment by thanateros** [comment]
Agreed; its a valid request -- frustrating to clone it for a new development project just to find it broken and not working.  My big issue is seeding; I get 'config/db/seeds/main.go:33:2: cannot find package "github.com/qor/qor-example/config/admin" in any of:' blah blah blah.  I respect the work put into it and I am enjoying using Qor better than any CMS from my past; however, at times like this I wonder what the wisdom is of pushing out an example site (that one expects to help get other developers on-board with Qor) that was, apparently, never even tested once.  I hope it gets fixed soon.  Looks like the entire admin folder is missing from config/ .

**Comment by alexmironof** [comment]
@jinzhu congrats! QOR attracted already many people 👍 🎉
